src/components/PageHero.tsx

```diff
@@ -0,0 +1,108 @@
+import styled from '@emotion/styled';
+
+import { FC } from 'react';
+import { Heading } from '~/components';
+import {
+  atMinLg,
+  atMinDesktop,
+  atMinLargeDesktop,
+  bp,
+  BreakpointName,
+} from '~/theme/breakpoints';
+import { cssClamp } from '~/theme/util';
+
+type HeroImageSource = {
+  largeDesktop: string;
+  tablet: string;
+  xl: string;
+  xs: string;
+};
+
+type HeroImageProps = {
+  alt: string;
+  imgSource: HeroImageSource;
+};
+
+type PageHeroProps = HeroImageProps & {
+  text: string;
+};
+
+const calculatedImageHeight = cssClamp(
+  [17.5, 'smMobile'],
+  [20.188, 'mobile'],
+  [35, 'tablet'],
+);
+
+const Image = styled.picture`
+  margin-left: calc(50% - 50vw);
+
+  ${atMinLargeDesktop} {
+    margin-left: ${cssClamp([-20, 'largeDesktop'], [-20, 'xlDesktop'])};
```

Comment:
🔧 No need for the clamp if the values aren't change. However, looking over the comps it does look like the image was too far to the left on the xlDesktop size.

```suggestion
    margin-left: ${cssClamp([-20, 'largeDesktop'], [-15, 'xlDesktop'])};
```

Reply:
Yeah, that's completely dumb on my part. Thanks for catching that!